pkg/java/pom/artifact.go

```diff
@@ -43,12 +44,26 @@ func (a artifact) Equal(o artifact) bool {
 	return a.GroupID == o.GroupID || a.ArtifactID == o.ArtifactID || a.Version.String() == o.Version.String()
 }
 
+func (a artifact) JoinLicenses() string {
+	return strings.Join(a.Licenses, ", ")
+}
+
+func (a artifact) ToPomLicenses() pomLicenses {
+	return pomLicenses{License: lo.Map(a.Licenses, func(lic string, _ int) pomLicense {
+		return pomLicense{Name: lic}
+	})}
+}
+
 func (a artifact) Inherit(parent artifact) artifact {
 	// inherited from a parent
 	if a.GroupID == "" {
 		a.GroupID = parent.GroupID
 	}
 
+	if len(a.Licenses) == 0 {
```

Comment:
I am not sure about this.
Do we need to get licenses from each parents?

Example:
We have `base pom` (contains `Apatch-1.0` license) with `package1`(uses `Apatch-1.0`) and `package2`(uses `Apatch-1.0`). `parent` also contains `Apatch1.0` (`package2` in `dependencyManagement`).
![image](https://github.com/aquasecurity/go-dep-parser/assets/91113035/d8b05a67-9d4f-49d1-a8b2-843aa9c0fd86)

After a while, `another-group` (parent pom) updates version of `package2`(uses `Apatch-2.0`) and updates `licenses` field in parent pom.
Owner of `base pom` just updated parent version:
![image](https://github.com/aquasecurity/go-dep-parser/assets/91113035/23d96c0b-871f-4c34-880d-31c7d5bdf6fc)

In this case i see 2 options:
1) This is mistake of `bases pom` owner - he should update licenses when updating parent version.
2) We need to inherit license from parent and include 2 licenses in result

@nikpivkin @knqyf263 wdyt?

Reply:
@DmitriyLewen the license field is overwritten in the child pom.xml. I was guided by the output of the [help:effective-pom](https://maven.apache.org/plugins/maven-help-plugin/effective-pom-mojo.html) command.

Reply:
I didn't know `help:effective-pom`. It looks really useful.

Reply:
okay, thanks for information. Let's stay this solution.